### ISSUE TITLE
Using repeated language tags doesn't work

### ISSUE THREAD

**Issue by sastels** [cds-snc/notification-planning#491]
## Describe the bug 

Repeated use of a language tag in a template doesn't work.

- [ ] timeboxed investigation 2-3 hours for 1 dev to try and find the cause of the bug

## [Bug Severity](https://docs.google.com/document/d/1tcD32bjJRaREsMgaUEoXAk400y1dKzzaKR0G5Sr1w20/edit)
SEV3 perhaps? messages can still be sent out, but screen readers might read parts incorrectly.

## To Reproduce

1. Create a template with content
```
[[fr]]
Le français suis l'anglais
[[/fr]]

[[en]]
hi
[[/en]]

[[fr]]
Bonjour
[[/fr]]
```
2. Save the template and send it to yourself. The second French tags are treated as text instead of being parsed and turned into html language tags.
```
<div lang="fr-ca">
    <p style="Margin:0 0 20px 0;font-size:19px;line-height:25px;color:#0b0c0c">
        Le français suis l’anglais
    </p>
</div>
<div lang="en-ca">
    <p style="Margin:0 0 20px 0;font-size:19px;line-height:25px;color:#0b0c0c">
        hi
    </p>
</div>
<p style="Margin:0 0 20px 0;font-size:19px;line-height:25px;color:#0b0c0c">
    [[fr]]<br>Bonjour<br>[[/fr]]
</p>
```

## Expected behavior

All tags are parsed into html language tags.

## Impact

Just had one user bring it to our attention, likely because not many people are using the language tags. But anyone trying to do things properly (ie starting with "Le français suis l'anglais") will have this issue.

Impact on Notify users:
- cannot use correct language tags

Impact on Recipients:
- screen readers might not read content correctly.

## Additional context

https://cds-snc.freshdesk.com/a/tickets/9024
https://cds-snc.freshdesk.com/a/tickets/9375
https://cds-snc.freshdesk.com/a/tickets/9388

## Acceptance Criteria
- [ ] Basic pattern of repeated language tags work (i.e. FR followed by EN followed by FR)
- [ ] Nested language tags work (i.e FR with EN inside of it followed by EN)
- [ ] Spot check of 10 most recently sent templates render correctly

**Comment by yaelberger-commits** [comment]
Validate priority level with @amazingphilippe

**Comment by ramya-cds** [comment]
Adding Freshdesk tickets here for tracking purpose - 
https://cds-snc.freshdesk.com/a/tickets/9024
https://cds-snc.freshdesk.com/a/tickets/9375

**Comment by amazingphilippe** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Notify-Planning-614b3ad91bc2030015ed22f5/issues/cds-snc/notification-planning/491?planning-poker) with ZenHub @andrewleith @jimleroyer @jzbahrai @sastels

**Comment by yaelberger-commits** [comment]
Ready to merge after code freeze.

**Comment by yaelberger-commits** [comment]
Steve M and Jimmy will QA today

**Comment by smcmurtry** [comment]
I QA'd with a screenreader and the bug appears to be fixed! I checked both sent email and the email preview.

Original Bug: https://youtube.com/shorts/o9hvwjgowxI
Fixed after applying the code in this PR: https://youtube.com/shorts/U-io4hg8TZ0

**Comment by yaelberger-commits** [comment]
Flag for me when complete and I will notify clients who reached out about this issue

**Comment by yaelberger-commits** [comment]
PRs ready to be released this morning

**Comment by jimleroyer** [comment]
Deployed in production.

**Comment by yaelberger-commits** [comment]
@jimleroyer Can I let the users know about the fix then and move this to done after that?

**Comment by jimleroyer** [comment]
@yaelberger-commits yes absolutely!

**Comment by yaelberger-commits** [comment]
I responded to all three support tickets about this bug that it's been fixed. Closing the ticket
